test/cli/test_cli_functions.sh

```diff
@@ -322,3 +321,61 @@ function check_deletes {
     delete_backingstore_path
     echo_time "✅  delete cycle is done"
 }
+
+function crd_arr { 
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
+    echo_time "${crd_array[*]}"
+}
+
+function noobaa_uninstall {
+    check_cleanflag=$((RANDOM%2))
+    echo_time ${check_cleanflag}
+    
+    if [ ${check_cleanflag} -gt 0 ] 
+    then
+        echo_time "Running uninstall with --cleanup”"
+        test_noobaa uninstall --cleanup 
+        check_if_cleanup
+    else
+        echo_time "Running uninstall without --cleanup”"
+        test_noobaa uninstall 
+    fi  
+}
+
+function check_if_cleanup {  
+    crd_array_after_Cleanup=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))   
+    for should_be_empty_crd_array in ${crd_array[@]}
+    do
+        for crd_after_clean in "${crd_array_after_Cleanup[@]}"
+        do
+            if [[ ${should_be_empty_crd_array} == ${crd_after_clean} ]]
+            then
+                echo_time "${should_be_empty_crd_array} is in crd"
+                exit 1   
+            else         
+                echo_time "${should_be_empty_crd_array} is not in crd, deleted with clenaup"
+            fi               
+        done
+    done
```

Comment:
we can check that the crd before the cleanup is not in the array after the cleanup
```suggestion
    for crd_before_clean in ${crd_array[@]}
    do
            if [[ ${crd_array_after_Cleanup[@]} =~ ${crd_before_clean} ]]
            then
                echo_time "${crd_before_clean} is in crd"
                exit 1   
            else         
                echo_time "${crd_before_clean} is not in crd, deleted with clenaup"
            fi               
    done
```